cmd/call-endpoint/main.go

```diff
@@ -45,7 +45,7 @@ func main() {
 		return
 	}
 
-	apiDescription, err := api.Describe(*api.GetRoutes())
+	apiDescription, err := core.Describe(*api.GetRoutes())
```

Comment:
Now that this function is moved, is needs a more descriptive name.
DescribeRoutes()?